src/Spice86/ViewModels/DisassemblyViewModel.cs

```diff
@@ -154,6 +158,23 @@ private static List<CpuInstructionInfo> DecodeInstructions(State state, IMemory
 
         return instructions;
     }
+    
+    /// <summary>
+    /// Handles a breakpoint being hit.
+    /// </summary>
+    /// <param name="breakPoint">The <see cref="BreakPoint"/> object representing the breakpoint that was hit.</param>
+    private void OnBreakPointReached(BreakPoint breakPoint) =>
+        _pauseHandler.RequestPause($"breakpoint {breakPoint.BreakPointType} hit");
+
+    [RelayCommand]
+    private void CreateUnconditionalBreakpoint() {
+        if (SelectedInstruction is null) {
+            return;
+        }
+        AddressBreakPoint breakPoint = new(BreakPointType.EXECUTION, SelectedInstruction.Address, OnBreakPointReached, false);
+        _emulatorBreakpointsManager.ToggleBreakPoint(breakPoint, true);
+        SelectedInstruction.HasBreakpoint = _emulatorBreakpointsManager.IsAddressBreakpointAt(SelectedInstruction.Address);
```

Comment:
Since you just created a breakpoint at this address, why not just true?

Reply:
Corrected.